File: hw1/src/test/java/CalculatorTest.java
import org.junit.Assert;
import org.junit.Test;

public class CalculatorTest {

    @Test
    public void Sum_2Plus5_7Returned() {
        var res = Calculator.calculate(2, "+", 5);
        Assert.assertEquals(7, res);
    }

    @Test
    public void calc_2Multiply5_10Returned() {
        var res = Calculator.calculate(2, "*", 5);
        Assert.assertEquals(10, res);
    }

    @Test
    public void calc_7Minus5_2Returned() {
        var res = Calculator.calculate(7, "-", 5);
        Assert.assertEquals(2, res);
    }

    @Test
    public void calc_6Divide3_2Returned() {
        var res = Calculator.calculate(6, "/", 3);
        Assert.assertEquals(2, res);
    }

    @Test
    public void calc_default() {
        var res = Calculator.calculate(6, ":", 3);
        Assert.assertEquals(0, res);
    }
}

File: hw2/src/test/kotlin/CalculatorTest.kt
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class CalculatorTest {

    private val calculator = Calculator()

    @Test
    fun calc_2Plus5_7Returned() {
        val res: Int = calculator.calculate(2, "+", 5)
        Assertions.assertEquals(7, res)
    }

    @Test
    fun calc_2Multiply5_10Returned() {
        val res: Int = calculator.calculate(2, "*", 5)
        Assertions.assertEquals(10, res)
    }

    @Test
    fun calc_7Minus5_2Returned() {
        val res: Int = calculator.calculate(7, "-", 5)
        Assertions.assertEquals(2, res)
    }

    @Test
    fun calc_6Divide3_2Returned() {
        val res: Int = calculator.calculate(6, "/", 3)
        Assertions.assertEquals(2, res)
    }

    @Test
    fun calc_default() {
        val res: Int = calculator.calculate(6, ":", 3)
        Assertions.assertEquals(0, res)
    }
}